lib/cli/cliObjects.js

```diff
@@ -292,15 +292,15 @@ module.exports = class CLIObjects extends CLICommand {
 
     /**
      * Delete all object from list sequentially
-     * @param objects {object} class
-     * @param ids {array} IDs
-     * @param callback {function} function() {}
+     * @param {object} objects class
+     * @param {string[]} ids IDs
+     * @param {() => void} callback
      */
     _deleteObjects(objects, ids, callback) {
         if (!ids || !ids.length) {
             callback && callback();
         } else {
-            const id = ids.pop;
+            const id = ids.pop();
```

Comment:
This had me confused for a while, because I didn't see the missing () at first. But the type was a function.